File: problems/2525-categorize-box-according-to-criteria/solution.c
char* categorizeBox(int length, int width, int height, int mass) {
    long long volume = (long long)length * width * height;
    int bulky = (length >= 10000 || width >= 10000 || height >= 10000 || volume >= 1000000000);
    int heavy = (mass >= 100);
    if (bulky && heavy) return "Both";
    if (bulky) return "Bulky";
    if (heavy) return "Heavy";
    return "Neither";
}

File: problems/2525-categorize-box-according-to-criteria/solution.py
class Solution:
    def categorizeBox(self, length: int, width: int, height: int, mass: int) -> str:
        bulky = False
        heavy = False
        volume = length * width * height
        if length >= 10000 or width >= 10000 or height >= 10000 or volume >= 1000000000:
            bulky = True
        if mass >= 100:
            heavy = True

        if bulky and heavy:
            return "Both"
        elif bulky:
            return "Bulky"
        elif heavy:
            return "Heavy"
        else:
            return "Neither"